config/routes.rb

```diff
@@ -30,8 +30,12 @@
         # 🛠 Product owner controls for managing tips
         resources :reflection_tips, only: [ :create, :destroy, :update, :show ]
 
-        resources :organizations, only: [ :create, :index, :show ]
```

Comment:
Since an OrganizationsController had not yet been implemented in the application, I opted to remove the existing resources :organizations route and fully implement resources :orgs instead, aligning the route structure more closely with the needs and workflows of product owners.